### PR TITLE
fix(BackTop): should hide when deactivated

### PR DESCRIPTION
**Pull request by zqran**
The `floating-bubble` component is mentioned here because I found this bug when reviewing this component.

After the https://github.com/youzan/vant/pull/11880 is merged in the future, no additional changes are required.

---

before:

https://github.com/youzan/vant/assets/15389209/10ded93c-3ad3-4f25-ba9e-f56d41c0cef9

after:

https://github.com/youzan/vant/assets/15389209/31ed949c-131f-4a5e-a97b-8b99371fdc22

### CODE REVIEW

File: packages/vant-cli/site/mobile/App.vue [hunk 1]
```diff
@@ -2,23 +2,34 @@
...
 
 <script>
-import { watch } from 'vue';
+import { watch, computed } from 'vue';
 import DemoNav from './components/DemoNav.vue';
 import { useCurrentTheme } from '../common/iframe-sync';
 import { config } from 'site-mobile-shared';
+import { router } from './router';
+
+const excludePath = ['floating-bubble', 'back-top'];
```
Comment (chenjiahan) [inline packages/vant-cli/site/mobile/App.vue:20]: Vant CLI 不是和 Vant 耦合的，它也会用于构建除了 Vant 以外的组件库。

因此我们不应该再 Vant CLI 里写死 Vant 组件的名称。

Comment (chenjiahan) [reply packages/vant-cli/site/mobile/App.vue:20]: 这个场景最好是在组件内适配，比如像 Popup 一样使用 onDeactivated

Comment (zqran) [reply packages/vant-cli/site/mobile/App.vue]: > 这个场景最好是在组件内适配，比如像 Popup 一样使用 onDeactivated

Get!

**Review by chenjiahan — APPROVED** [review]

**Comment by chenjiahan** [comment]
👍🏻

**Pull request merged**